src/idl_gen_python.cpp

```diff
@@ -551,7 +551,30 @@ class PythonGenerator : public BaseGenerator {
 
     GetEndOffsetOnTable(struct_def, code_ptr);
   }
-
+  
+  // Generate funtion to cehck for propper file identifier
+  void GenHasFileIdentifier(const StructDef &struct_def,
+                             std::string *code_ptr) {
+    std::string &code = *code_ptr;
+    std::string esapedID;
+    //in the event any of file_identifier characters are special(NULL, \, etc), problems occur.
+    //to prevent this, convert all chars to there hex escaped equivilent
+    for(int x=0; x<4; x++){
+		esapedID += "\\x" + IntToStringHex(parser_.file_identifier_[x], 2);
```

Comment:
I guess this can stay in.. if we want the `file_indentifier` to not contain these character, that's the responsiblity of the schema compiler, not the code generator.

Reply:
@pyottamus @aardappel Do you have consensus on this string encoding stuff?